core/src/test/scala/org/apache/spark/benchmark/Benchmark.scala

```diff
@@ -158,7 +159,8 @@ private[spark] class Benchmark(
     // scalastyle:on
     val best = runTimes.min
     val avg = runTimes.sum / runTimes.size
-    Result(avg / 1000000.0, num / (best / 1000.0), best / 1000000.0)
+    val stdev = math.sqrt(runTimes.map(time => math.pow(time - avg, 2)).sum / runTimes.size)
```

Comment:
Not that it really matters, but `(time - avg) * (time - avg)` is fine here and faster than pow.
Super nit but I'd suggest it's more reasonable to use the sample rather than population stdev: divide by `runTimes.size - 1`. I suppose this means also checking that there are at least 2 runs.

Reply:
Ah ok, I agree with you on both. If there aren't enough runs, should we just put "N/A" then?

Reply:
You can assume (or assert) that there was at least 1 benchmarking run, or none of the metrics mean anything. (maybe it's already asserted)

While the sample stdev is not really defined for 1 run, "0" is fine.

Reply:
I don't think it's asserted anywhere. i'll add it